### PR TITLE
en: add whats new in v1.3

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

- Add whats new in v1.3.
- Delete whats new in v1.2/v1.1

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1640
- Other reference link(s):

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/whats-new-in-v1.3.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# What's New in TiDB Operator 1.3
+
+TiDB Operator 1.3 introduces the following key features, which helps you manage TiDB clusters and the tools more easily in terms of extensibility, usability, and security.
+
+## Compatibility changes
+
+- Enhance the feature of deploying a TiDB cluster across Kubernetes clusters. If you deployed a TiDB cluster across multiple Kubernetes clusters by TiDB Operator (<= v1.3.0-beta.1), upgrading TiDB Operator to v1.3.0 directly will cause failed rolling upgrade and the cluster might become abnormal. If you need to upgrade TiDB Operator from earlier versions to v1.3.0, take the following steps:
+
+    1. Update CRD.
+    2. Add a new `spec.acrossK8s` field in TidbCluster spec and set it to `true`.
```
Comment (qiancai) [inline en/whats-new-in-v1.3.md:14]:
```suggestion
    2. Add a new `spec.acrossK8s` field in the TidbCluster spec and set it to `true`.
```

File: en/whats-new-in-v1.3.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+title: What's New in TiDB Operator 1.3
+---
+
+# What's New in TiDB Operator 1.3
+
+TiDB Operator 1.3 introduces the following key features, which helps you manage TiDB clusters and the tools more easily in terms of extensibility, usability, and security.
+
+## Compatibility changes
+
+- Enhance the feature of deploying a TiDB cluster across Kubernetes clusters. If you deployed a TiDB cluster across multiple Kubernetes clusters by TiDB Operator (<= v1.3.0-beta.1), upgrading TiDB Operator to v1.3.0 directly will cause failed rolling upgrade and the cluster might become abnormal. If you need to upgrade TiDB Operator from earlier versions to v1.3.0, take the following steps:
```
Comment (qiancai) [inline en/whats-new-in-v1.3.md:11]:
```suggestion
- Enhance the feature of deploying a TiDB cluster across Kubernetes clusters. If you deploy a TiDB cluster across multiple Kubernetes clusters by TiDB Operator (<= v1.3.0-beta.1), upgrading TiDB Operator to v1.3.0 directly will cause failed rolling upgrade and the cluster might become abnormal. If you need to upgrade TiDB Operator from earlier versions to v1.3.0, take the following steps:
```

File: en/whats-new-in-v1.3.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+- TiKV under specific configurations might be rolling updated. If you deploy TiDB v5.0 or later versions, and configure `spec.tikv.seperateRocksDBLog: true` or `spec.tikv.separateRaftLog: true`, upgrading TiDB Operator to v1.3.x will cause TiKV to be rolling updated.
+
+- To support configuring Prometheus shards in the TidbMonitor CR, upgrading TiDB Operator to v1.3.x will cause the TidbMonitor Pod to be rolling updated.
+
+## Extensibility
+
+- [Deploying a TiDB cluster across multiple Kubernetes clusters](deploy-tidb-cluster-across-multiple-kubernetes.md) becomes generally available. Now you can deploy [heterogeneous TiDB clusters](deploy-heterogeneous-tidb-cluster.md) across Kubernetes clusters.
+- Add a new `failover.recoverByUID` field to support one-time recover for TiKV/TiFlash/DM Worker.
```
Comment (qiancai) [inline en/whats-new-in-v1.3.md:30]:
```suggestion
- Add a new `failover.recoverByUID` field to support one-time recovery for TiKV, TiFlash, and DM Worker.
```

File: en/whats-new-in-v1.3.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+- Support configuring [PodManagementPolicy](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies) for the StatefulSets of TiDB cluster components.
+- Support configuring DNS config for Pods of all components.
```
Comment (qiancai) [inline en/whats-new-in-v1.3.md:32]:
```suggestion
- Support configuring DNS for Pods of all components.
```

**Review by qiancai — COMMENTED** [review]
@ran-huang PTAL

File: en/whats-new-in-v1.3.md (hunk 4 above)
Comment (ran-huang) [reply en/whats-new-in-v1.3.md:32]: 这里之前我也这样改了，研发说“配置 DNS 配置“和“配置 DNS”不一样😂
改成 “configuring DNS settings”？
image

Comment (qiancai) [reply en/whats-new-in-v1.3.md:32]: 好的，可能研发想强调不是手动配的 DNS 而是通过配置文件配置的吗？

**Review by qiancai — APPROVED** [review]

**Pull request merged**